IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -8904,6 +8904,7 @@
 					"$(inherited)",
 					"$(MSID_WEBKIT)",
 					"$(MSID_SYSTEMWV)",
+					"$(MSAL_JS_AUTOMATION)",
```

Comment:
Just curious, will javascript injection be available both on iOS and macOS?

Reply:
I didn't try it on macOS since the current focus is only for iOS, but I suppose it should work.